docs/stdlib/set.rst

```diff
@@ -393,14 +400,14 @@ Sets
 
     :index: cardinality singleton
 
-    Check that the input set contains no more than one element.
+    Returns :eql:type:`anytype` based on condition of if the set is singular.
```

Comment:
```suggestion
    Checks that the input set contains no more than one element.
```

Reply:
Like I mentioned about trying to standardize language around operators, I'd like to do the same around functions. In most of what's rewritten in this PR, we're trying to use "Returns" in the one-liners to talk about what comes out of a function.

All that said, what about this?

```rest
    Returns ``true`` if the given set contains no more than one element.
```

Reply:
```suggestion
    Returns ``true`` if the given set contains no more than one element.
```